pkg/store/experiment/experiment.go

```diff
@@ -25,7 +27,20 @@ func NewStore(db *dbstore.DB) core.ExperimentStore {
 	return &experimentStore{db}
 }
 
-// TODO: need to define the fields of this struct
 type experimentStore struct {
 	db *dbstore.DB
 }
+
+// TODO: implement core.EventStore interface
```

Comment:
I think we need a `delete` interface here as well. Do you want to put it in later prs?